### DISCUSSION TITLE
Any alternatives to AllocConsole but for the new terminal?

### DISCUSSION THREAD

**Discussion by slxdy** [microsoft/terminal#18078] [Q&A]
I have an application and I'd like to start a new console window in the new terminal instead of the old conhost window (AllocConsole). Is this possible?

**Comment by lhecker** [comment]
On recent versions of Windows you can set Windows Terminal as your default terminal. `AllocConsole` should then open a new tab in Windows Terminal. Are you on Windows 10? You can opt into it by updating to the latest version and enabling it here:

![image](https://github.com/user-attachments/assets/8fa0c234-bda8-4986-afee-502f9500186f)

Don't forget to click "Save" after changing the option.

**Reply by slxdy** [reply]
Yes, I know about that option, but AllocConsole completely ignores it.

**Reply by GrinlexGH** [reply]
> Yes, I know about that option, but AllocConsole completely ignores it.

any fixes? I have the same problem

**Reply by tom-huntington** [reply]
doesn't work for me

**Reply by Meigyoku-Thmn** [reply]
It's 2026 and the AllocConsole WinAPI still doesn't work with Windows Terminal.
